src/Foundatio.Repositories.Elasticsearch/Queries/Builders/FieldIncludesQueryBuilder.cs

```diff
@@ -74,8 +77,11 @@ public static class FieldIncludesQueryExtensions {
         }
 
         public static T ExcludeFields<T>(this T query, params Field[] fields) where T : IFieldIncludesQuery {
-            if (fields != null)
-                query.FieldExcludes?.AddRange(fields);
+            if (fields == null)
+                return query;
+
+            foreach (var field in fields.Where(p => !string.IsNullOrEmpty(p.Name)))
```

Comment:
Yeah, you are right, we can't just check Name. Name isn't populated if you use a lambda expression. So not sure what all you would have to check to see if its empty.